src/scenes/ObservationsRouter/ObservationsHome.tsx

```diff
@@ -43,9 +40,8 @@ export default function ObservationsHome(props: ObservationsHomeProps): JSX.Elem
   const { selectedOrganization } = useOrganization();
   const [selectedPlantingSite, setSelectedPlantingSite] = useState<PlantingSite>();
   const [plantsSitePreferences, setPlantsSitePreferences] = useState<Record<string, unknown>>();
-  const [view, setView] = useState<View>();
+  const [view] = useState<View>();
   const plantingSites = useAppSelector(selectPlantingSites);
```

Comment:
Is `view` used anymore in this file, or can we get rid of this `setState`?

Reply:
It is used in line 130, but probably it is not necessary anymore. Let me take another look